pkg/vm/memory/relocatable.go

```diff
@@ -229,11 +229,8 @@ func (m MaybeRelocatable) Sub(other MaybeRelocatable) (MaybeRelocatable, error)
 		return *NewMaybeRelocatableRelocatable(relocatable), nil
 
 	} else if is_rel_m && is_rel_other {
-		offset_diff, err := m_rel.Sub(other_rel)
-		if err != nil {
-			return *NewMaybeRelocatableFelt(lambdaworks.FeltZero()), err
-		}
-		return *NewMaybeRelocatableFelt(lambdaworks.FeltFromUint64(uint64(offset_diff))), nil
+		offset_diff := lambdaworks.FeltFromUint64(uint64(m_rel.Offset)).Sub(lambdaworks.FeltFromUint64(uint64(other_rel.Offset)))
+		return *NewMaybeRelocatableFelt(offset_diff), nil
```

Comment:
Don't change this, the bug is in `Relocatable.Sub` and will pop up later if left unfixed

Reply:
This fix should go to `Relocatable.Sub`